nixos/modules/config/system-path.nix

```diff
@@ -102,7 +122,7 @@ in
 
   config = {
 
-    environment.systemPackages = requiredPackages;
+    environment.systemPackages = requiredPackages ++ defaultPackages;
```

Comment:
Should not this be `config.environment.defaultPackages`?

Reply:
sure. good catch!